dash/frontend/src/app/modules/private/private-routing.module.ts

```diff
@@ -313,275 +313,274 @@ const routes: Routes = [
+        },
+        children: [
+          {
+            path: '',
+            component: DockerRegistriesListComponent,
+            data: {
+              title: 'Docker Registries',
+            }
+          },
+          {
+            path: 'create',
+            component: DockerRegistriesCreateComponent,
+            canActivate: [RoleGuard],
+            data: {
+              title: 'Create Docker Registries',
+              allowedUserRoles: [Authority.SUPER_ADMIN, Authority.ADMIN]
+            }
+          },
+          {
+            path: 'edit/:id',
+            component: DockerRegistriesCreateComponent,
+            canActivate: [RoleGuard],
+            data: {
+              title: 'Edit Docker Registry',
+              allowedUserRoles: [Authority.SUPER_ADMIN, Authority.ADMIN]
+            }
+          }
+        ]
       },
       {
-        path: 'create',
-        component: ApiKeyFormComponent,
-        canActivate: [RoleGuard],
+        path: 'api-key',
+        component: OrganizationSettingsComponent,
+        canActivateChild: [RoleGuard],
         data: {
-          title: 'API Key Management',
           allowedUserRoles: [Authority.SUPER_ADMIN]
-        }
+        },
+        children: [
+          {
+            path: '',
+            component: ApiKeyListComponent,
+            canActivate: [RoleGuard],
+            data: {
+              title: 'API Key Management',
+            }
+          },
+          {
+            path: 'create',
+            component: ApiKeyFormComponent,
+            canActivate: [RoleGuard],
+            data: {
+              title: 'API Key Management',
+              allowedUserRoles: [Authority.SUPER_ADMIN]
+            }
+          },
+          {
+            path: 'edit/:id',
+            component: ApiKeyFormComponent,
+            canActivate: [RoleGuard],
+            data: {
+              title: 'Update Key Management',
+              allowedUserRoles: [Authority.SUPER_ADMIN]
+            }
+          },
+        ]
       },
       {
-        path: 'edit/:id',
-        component: ApiKeyFormComponent,
-        canActivate: [RoleGuard],
+        path: 'audit-logs',
+        component: OrganizationSettingsComponent,
+        canActivateChild: [RoleGuard],
         data: {
-          title: 'Update Key Management',
           allowedUserRoles: [Authority.SUPER_ADMIN]
-        }
-      },
-    ]
-  },
-  {
-    path: 'audit-logs',
-    component: OrganizationSettingsComponent,
-    canActivateChild: [RoleGuard],
-    data: {
-      allowedUserRoles: [Authority.SUPER_ADMIN]
-    },
-    children: [
-      {
-        path: '',
-        component: AuditLogComponent,
-        canActivate: [RoleGuard],
-        data: {
-          title: 'Audit Logs',
-        }
+        },
+        children: [
+          {
+            path: '',
+            component: AuditLogComponent,
+            canActivate: [RoleGuard],
```

Comment:
Did anything need to be updated in RoleGuard?

Reply:
No. RoleGuard looks at the currently-logged-in user's permissions.